### DISCUSSION TITLE
Add alarm as an option besides the reminder to remind about your tasks

### DISCUSSION THREAD

**Discussion by NotSesmic** [iSoron/uhabits#1171] [Feature Requests]
Can you add alarm as an option as I find it better when it comes to reminding about tasks and I use alarms to wake me up too (I know I can use clock app to set an alarm 😛)

Thank you :)

**Comment by hiqua** [comment]
Do you mean that you find the current notifications not noisy enough?

**Reply by NotSesmic** [reply]
> Do you mean that you find the current notifications not noisy enough?

You can say that 😂

**Reply by hiqua** [reply]
Not sure if that's technically possible, but it seems like the solution would be to allow users to set an optional ringtone for reminders.

Honestly I'm not sure a lot of users would be interested in that, and you can already set your own alarms if that's really important for you.

**Reply by MIbrahimPro** [reply]
It would actually be better, 
if an alarm goes of from alarm app, you are only reminded not set it to true or false
but the alarm from this app could have the notes and texts and allow to snooze and even set it to true or false (✔️ or ❌)
